<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mdi</class>
 <widget class="QMainWindow" name="Mdi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>24</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>25</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>26</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>27</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>28</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>29</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>30</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>备注</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="cobwidget" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>无</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A  区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-1区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-2区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-3区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-4区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-5区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-6区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C  区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D  区域</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S  区域</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/saveAs.png</normaloff>:/img/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>打开现有文件</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
